### ISSUE TITLE
Update court info methods due to there now being multiple Ottawa Circuit Court addresses

### ISSUE THREAD

**Issue by ekressmiller** [mplp/docassemble-mlhframework#152]
From Ottawa County person:
"I noticed that the address for Ottawa County is incorrect. With our move, all domestic forms should now have the address: 12240 Fillmore St, Rm 50, West Olive, MI 49460; phone: 616-846-8315"

From Charlie, in October email thread, after following up for more details:
"all family issues (DM, DO, DC new cases and all post-judgment family motions) are filed at the new courthouse in West Olive and all other filings are done at the Grand Haven address (including Name Changes, PPOs, all criminal, and foreclosure proceeds filings). I'm also not sure what address we have on the small estate tool for Ottawa County. Part of this transition is that the Probate Court moved to Grand Haven so the small estate tool should provide that address if it doesn't now. This is super confusing so let me know if we need additional follow-up."

specifically to do:
- update probate address to "414 Washington Ave., Grand Haven, MI 49417"
- [x] in the Google sheet 
- [x] notify Eli of update for MLH site (MLH site already updated actually)
- [x] in Docassemble
- [ ] in LHI
- update circuit court contact info for "family issues" versus other case types per documentation above such that all family/domestic stuff has the West Olive address, and everything else has the Grand Haven address.
- [x] figure out what to do about this split in the Google sheet to make it most friendly to existing Docassemble address system (and do it)
- [x] notify Eli of update for MLH site
- [ ] in Docassemble
- [ ] in LHI
- FOC also to be updated to West Olive address, and phone (877) 543-2660
- [x] in Google sheet
- [x] notify Eli of update for MLH site
- [x] in Docassemble
- [ ] in LHI
- make sure all references to the Grand Haven address are to Washington **Ave.** not Washington St.
- [x] in Google sheet
- [x] in Docassemble
- [ ] in LHI
- district court info should NOT need to change

**Comment by normon66** [comment]
@ekressmiller - is there an update here that needs to and is ready to be implemented in the rest of the MLH portfolio?

**Comment by ekressmiller** [comment]
@normon66 I'm waiting to get confirmation from Charlie on a couple details about the change(s?), and will update you once I hear back.

**Comment by ekressmiller** [comment]
I pinged charlie again on 11/15.

**Comment by ekressmiller** [comment]
@normon66 see updated info above. Let me know if you have questions. We may need to follow up if we're not sure which way any of our case types fall on the family vs. everything else continuum

**Comment by ekressmiller** [comment]
As of 12/13, all changes made in Docassemble except the circuit court thing. Have reached out to Kim and Charlie about how to handle fee waiver or other forms that can be for multiple subject matter areas. Will decide how to program this in once have answer on that.

**Comment by normon66** [comment]
@ekressmiller - do you know which phone number should be used for the new Ottawa FOC location? The original details above show 616-846-8315, [their website](https://www.miottawa.org/Courts/20thcircuit/) shows (616) 786-4100, and the [Court Contact Google Sheet](https://docs.google.com/spreadsheets/d/1dpryBE_1tRBpf-i6TIsmAtGx9AuiBpFb6hTNCGUiWMI/edit?gid=0#gid=0) shows (877) 543-2660.

**Comment by ekressmiller** [comment]
note to self:

reupload court list with updated Ottawa circuit court (nondomestic) address. (removed n and add suite number)

**Comment by ekressmiller** [comment]
Yeah, @normon66, just to confirm the [phone number for the FOC](https://www.miottawa.org/Courts/FOC/) is different from the new circuit court family matters number.

**Comment by normon66** [comment]
Thanks, @ekressmiller! I think the fee waiver is the only tool that this will affect. There are some other tools that can be used for either circuit or district, but in those cases we always know what type of circuit case it would be (ie DM/DO/DC vs something else). I'll get that into my work queue.

**Comment by ekressmiller** [comment]
@normon66 I've started working on this for the fee waiver tool, though running into roadblocks.

**Comment by ekressmiller** [comment]
@normon66 maybe it's working now!?! Feel free to review [the PR](https://github.com/mplp/docassemble-FeeWaiverRequest/pull/67) if you have bandwidth, which depends on [this framework PR](https://github.com/mplp/docassemble-mlhframework/pull/161)

I ran into lots of "input not processed errors" in earlier versions of this, but I'm not triggering them now, which I'm hoping reflects that this actually works and not just that I'm missing them.

**Comment by ekressmiller** [comment]
Note to self, do another quick round of testing on the non-fee-waiver tools when merging framework changes, since additional changes were made (for the fee waiver) after testing those.

**Comment by normon66** [comment]
@ekressmiller - I'll take a little time to look at this over these next couple days!

**Comment by ekressmiller** [comment]
@normon66 OK, it looks like both the Monroe and Macomb additional addresses are defunct. The Monroe 3600 Custer Road one goes to their youth treatment center, and they confirmed they don't hear cases there. And the Macomb 21850 Dunham one is to an old building--the court has since moved. But the phone number is (586) 469-5290. I'm removing the old addresses from the google sheet and the Docassemble list.

**Comment by normon66** [comment]
@ekressmiller - thanks for letting me know. I will update the A2J fee waiver tool to match.

**Comment by ekressmiller** [comment]
Note to self: after Brett has a chance to review, can publish these. Kim already approved the design change to the fee waiver tool.
